src/main/kotlin/no/nav/familie/ef/sak/økonomi/ØkonomiKlient.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.familie.ef.sak.økonomi
+
+import no.nav.familie.ef.sak.common.BaseService
+import no.nav.familie.ef.sak.økonomi.dto.StatusFraOppdragDTO
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsoppdrag
+import no.nav.familie.log.NavHttpHeaders
+import no.nav.familie.log.mdc.MDCConstants
+import no.nav.security.token.support.client.core.oauth2.OAuth2AccessTokenService
+import no.nav.security.token.support.client.spring.ClientConfigurationProperties
+import org.slf4j.MDC
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.web.client.RestTemplateBuilder
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpMethod
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Service
+import org.springframework.web.client.exchange
+import java.net.URI
+import java.time.LocalDateTime
+
+private const val OAUTH2_CLIENT_CONFIG_KEY = "familie-oppdrag-clientcredentials"
+const val FAGSYSTEM = "EF"
+
+@Service
+class ØkonomiKlient(
+        @Value("\${FAMILIE_OPPDRAG_API_URL}")
+        private val familieOppdragUri: String,
+        restTemplateBuilderMedProxy: RestTemplateBuilder,
+        clientConfigurationProperties: ClientConfigurationProperties,
+        oAuth2AccessTokenService: OAuth2AccessTokenService
+) : BaseService(
```

Comment:
Bruk AbstractRestClient eller AbstractPingableRestClient i stedet for BaseService.